src/components/Alert/Alert.js

```diff
@@ -3,13 +3,15 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Button } from '../Button';
 import { Icon } from '../Icon';
-import { getClassName, getIconName } from './helpers';
+import { getClassName, getIconName, warnIfDeprecatedType } from './helpers';
 import { ALERT_TYPES, ALERT_TYPE_ERROR } from './constants';
 
 /**
  * Alert Component for Patternfly React
  */
 const Alert = ({ children, className, onDismiss, type, ...props }) => {
+  warnIfDeprecatedType(type);
+
```

Comment:
Should we auto-change the type from 'danger' to 'error' here?

Reply:
I believe not, the helpers should know how to translate `danger` into `className` and `icon`